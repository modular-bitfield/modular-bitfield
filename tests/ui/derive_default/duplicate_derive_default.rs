use modular_bitfield::prelude::*;

#[bitfield]
#[derive(Default)]
#[derive(Default)]
pub struct DuplicateDefault {
    flag: bool,
    value: B7,
}

fn main() {}