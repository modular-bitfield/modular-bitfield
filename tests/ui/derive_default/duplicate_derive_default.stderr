error: encountered duplicate `#[derive(Default)]` parameter
 --> tests/ui/derive_default/duplicate_derive_default.rs:5:10
  |
5 | #[derive(Default)]
  |          ^^^^^^^

error: previous `#[derive(Default)]` parameter here
 --> tests/ui/derive_default/duplicate_derive_default.rs:4:10
  |
4 | #[derive(Default)]
  |          ^^^^^^^
